client/containers/Pub/PubDocument/Review/ReviewModal.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from 'react';
+import { Button, NonIdealState } from '@blueprintjs/core';
+
+import { apiFetch } from 'client/utils/apiFetch';
+import { DialogLauncher } from 'components';
+import { DocJson } from 'types';
+import { usePageContext } from 'utils/hooks';
+
+import ReviewerDialog from './ReviewerDialog';
+
+type Props = {
```

Comment:
Let's please grab the appropriate types from the `types` directory — `Pub` and `Community` are really easy to import, and you might also be interested in the `Callback` type.

```ts
// For a callback with no arguments
type Callback = () => unknown;

// For a callback with an argument of type T
type Callback<T> = (t: T) => unknown;
```

In your case you could e.g. write `updatePubData: Callback<Partial<Pub>>`